anagrafica/templates/anagrafica_utente_contatti.html

```diff
@@ -174,7 +174,56 @@ <h4 class="panel-title">
 
                 </div>
 
+                <a name="gsuite"></a>
 
+                <div class="panel panel-info">
+
+                    <div class="panel-heading">
+                        <h4 class="panel-title">
+                            <i class="fa fa-fw fa-envelope"></i>
+                            E-mail di servizio
+                            <span class="text-muted">(opzionale)</span>
+                        </h4>
+
+                    </div>
+                    {% if me.volontario %}
+                        {% if me.ha_email_servizio %}
+                            <div class="panel-body">
+                             <p>
+                                Questa è la tua e-mail dell'organizzazione. Potrai decidere dopo averla creata
+                                 se usarla come e-mail di contatto o come e-mail di login.
+                            </p>
+
+                            <pre>{{ me.email_servizio }}</pre>
+
+                            {% if errore_reset_email_servizio %}
+                                <span class="alert alert-warning">{{ errore_reset_email_servizio }}</span>
+                                <br />
+                            {% endif %}
+                            <a href="{% url 'anagrafica-utente_contatti' %}?reset_gsuite_password=1"
+                                class="btn btn-block btn-danger"
+                                onclick="return confirm('Vuoi procedere con il rest della password dell\'email di servizio?')">
+                                <i class="fa fa-reload fa-fw"></i>
+                                   Reset password
+                            </a>
+                        </div>
+                        {% else %}
+                            <div class="panel-body">
+                                <p>Vuoi attivare una e-mail di servizio?</p>
```

Comment:
Necessita di almeno una singola frase per spiegare cosa sia una email di servizio. @luca-dex?

Reply:
Tutta GAIA è stata realizzata in modo che ogni singola vista contenga le informazioni che permettono all'utente di capire cosa può fare, con rimandi ai relativi regolamenti. Scrivere _Vuoi attivare un'email di servizio?_ sicuramente ingenererà una serie di domande negli utenti perchè non si fa rimenrimento a nessun tipo di documentazione ralativa a questa situazione. È necessario spiegare cosa un Volontario può fare con un'email di servizio, quali sono i requisiti per richiederla e a quale regolamento viene fatto riferimento.